rflx/model.py

```diff
@@ -735,9 +751,9 @@ def __prove_overlays(self) -> None:
                         )
 
     def _prove(self) -> None:
-        self.__prove_contradictions()
         self.__prove_conflicting_conditions()
         self.__prove_reachability()
+        self.__prove_contradictions()
```

Comment:
What is the reason for changing the order?

Reply:
As I added path constraints to get the length of the field being compared with an aggregate into the facts, unreachable fields are now also detected as contradictions. However, I wanted to retain the more specific "field X is unreachable" message instead of a generic statement that there is a contradiction. Hence, I check contradictions after reachability now.